lib/puppet/pops/evaluator/closure.rb

```diff
@@ -25,9 +28,13 @@ def initialize(evaluator, model, scope)
   # Evaluates a closure in its enclosing scope after having matched given arguments with parameters (from left to right)
   # @api public
   def call(*args)
+    # We only want to see the global scope here. Other ephemeral scopes must be popped
```

Comment:
"We" - who is we in this context. Please rephrase - e.g. "only global scope should be visible here"...